src/components/Home/Hero.tsx

```diff
@@ -1,38 +1,33 @@
-import Image from 'next/image';
-import React from 'react';
-import bikeWoman1 from '../../assets/images/bike-woman1.png';
+import Image from "next/image";
+import React from "react";
+import Link from "next/link";
+import bikeWoman1 from "../../assets/images/bike-woman 1.png";
```

Comment:
I know this isn't part of the ticket but do you think we could clean up the names of the images so they don't have spaces in them? Just don't want this to cause problems in production

Reply:
To add this, I was checking and playing around with latest development branch. I encountered the issue that image name is not correct and gave me an error.  There are also more image names with space.

Also, I noticed there are some duplicated images, maybe you could clean up while you are at this?

Reply:
I would suggest we stop messing around with the image file names until the user preferences PR gets merged. Only because I made some fixes there that related to this. We can make a last PR that can be a 'clean up' PR

Reply:
All right, I'll wait until this is updated to update the image files. But we do need to remove the spacing and make them uniform.